core/src/integration-test/java/com/scalar/db/storage/jdbc/JdbcAdminImportTestUtils.java

```diff
@@ -0,0 +1,442 @@
+    } else {
+      throw new RuntimeException();
+    }
+
+    // table with all supported columns
+    sqls.add(
+        prepareCreateTableSql(
+            namespace, "good_table", goodTableColumns, goodTableMetadata.getPartitionKeyNames()));
+    results.put("good_table", goodTableMetadata);
+
+    // tables with an unsupported column
+    badTables.forEach(
+        (table, sql) -> {
+          sqls.add(sql);
+          results.put(table, null);
+        });
+
+    createExistingDatabase(namespace);
+    execute(sqls.toArray(new String[0]));
+    return results;
+  }
+
+  public void createExistingDatabase(String namespace) throws SQLException {
+    execute(rdbEngine.createNamespaceSqls(rdbEngine.enclose(namespace)));
+  }
+
+  public void dropTable(String namespace, String table) throws SQLException {
+    String dropTable = "DROP TABLE " + rdbEngine.encloseFullTableName(namespace, table);
+    execute(dropTable);
+  }
+
+  public void execute(String sql) throws SQLException {
+    try (BasicDataSource dataSource = JdbcUtils.initDataSourceForAdmin(config, rdbEngine);
+        Connection connection = dataSource.getConnection()) {
+      JdbcAdmin.execute(connection, sql);
+    }
+  }
+
+  public void execute(String[] sql) throws SQLException {
+    try (BasicDataSource dataSource = JdbcUtils.initDataSourceForAdmin(config, rdbEngine);
+        Connection connection = dataSource.getConnection()) {
+      JdbcAdmin.execute(connection, sql);
+    }
+  }
+
+  private LinkedHashMap<String, String> prepareColumnsForMysql() {
+    LinkedHashMap<String, String> columns = new LinkedHashMap<>();
+    columns.put("pk1", "INT");
+    columns.put("pk2", "INT");
+    columns.put("col01", "BOOLEAN");
+    columns.put("col02", "INT");
+    columns.put("col03", "INT UNSIGNED");
+    columns.put("col04", "TINYINT");
+    columns.put("col05", "SMALLINT");
+    columns.put("col06", "MEDIUMINT");
+    columns.put("col07", "BIGINT");
+    columns.put("col08", "FLOAT");
+    columns.put("col09", "DOUBLE");
+    columns.put("col10", "CHAR(8)");
+    columns.put("col11", "VARCHAR(512)");
+    columns.put("col12", "TEXT");
+    columns.put("col13", "TINYTEXT");
+    columns.put("col14", "MEDIUMTEXT");
+    columns.put("col15", "LONGTEXT");
+    columns.put("col16", "VARBINARY(1024)");
+    columns.put("col17", "BLOB");
+    columns.put("col18", "TINYBLOB");
+    columns.put("col19", "MEDIUMBLOB");
+    columns.put("col20", "LONGBLOB");
+    columns.put("col21", "BINARY(255)");
+    if (JdbcEnv.isMariaDB()) {
+      columns.put("col22", "JSON");
+    }
+    return columns;
+  }
+
+  private TableMetadata prepareTableMetadataForMysql() {
+    TableMetadata.Builder builder =
+        TableMetadata.newBuilder()
+            .addColumn("pk1", DataType.INT)
+            .addColumn("pk2", DataType.INT)
+            .addColumn("col01", DataType.BOOLEAN)
+            .addColumn("col02", DataType.INT)
+            .addColumn("col03", DataType.BIGINT)
```

Comment:
Refer to the original data types above and how they are expected to be mapped to ScalarDB data types. We probably need a document for supported and unsupported data types.